tests/pyrog/api/test_column.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from faker import Faker
+
+from django.urls import reverse
+
+faker = Faker()
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.mark.parametrize("table, column_field, status_code", [(faker.word(), faker.word(), 201)])
+def test_create_column(
+    api_client,
+    join,
+    input_factory,
```

Comment:
why do you use `input_factory` instead of `input` here?

Reply:
pls don't tell me it's because input is a keyword in python 😨

Reply:
haha no ;)
In the `conftest.py`, we `register` the factories.
Essentially, it declares two fixtures for each factory. A `column` fixture and a `column_factory` fixture, for instance.
Internally, the `column` will use the `input` fixture (for the dependency), and *not the `input_factory` fixture!*
The `input` fixture will always be the same actual model *instance* for the lifetime of the scope.

So in the present case, if we used `input` rather than `input_factory`, we would have `column.input == input`. Which we dont want.

Reply:
alright, missed that!